project/versions.scala

```diff
@@ -16,13 +16,13 @@
 //: ----------------------------------------------------------------------------
 
 object V {
-  val http4sArgonaut61 = "0.16.0"
-  val http4s           = "0.18.9"
+  val http4sArgonaut61 = "0.20.0-M4"
```

Comment:
Is this used anymore?  We haven't published http4s-argonaut61 for a long time.

Reply:
Done